nylas/client/client.py

```diff
@@ -654,7 +659,7 @@ def _delete_resource(self, cls, id, data=None, **kwargs):
         else:
             _validate(session.delete(url))
 
-    def _put_resource(self, cls, id, data, extra=None, path=None, **kwargs):
+    def _setup_update_resource(self, cls, id, data, extra=None, path=None, **kwargs):
```

Comment:
`id` is a built-in function and you might now want to shadow it, same goes for a few functions below.
Consider using a more descriptive name, like `resource_id`

Reply:
Good call out, done ✅